docker/chronos-nightly/README.md

```diff
@@ -63,19 +71,19 @@ sudo docker build \
 sudo docker build \
     --build-arg http_proxy=http://<your_proxy_ip>:<your_proxy_port> \ #optional
     --build-arg https_proxy=http://<your_proxy_ip>:<your_proxy_port> \ #optional
-    -t chronos-nightly:b1 . # You may choose any NAME:TAG you want.
+    -t intelanalytics/bigdl-chronos:latest . # You may choose any NAME:TAG you want.
 ```
 According to your network status, this building will cost **15-30 mins**. 
 
 **Tips:** When errors happen like `failed: Connection timed out.`, it's usually related to the bad network status. Please build with a proxy.
 
 ## Run the image
 ```bash
-sudo docker run -it --rm --net=host chronos-nightly:b1 bash
+sudo docker run -it --rm --net=host intelanalytics/bigdl-chronos:latest bash
 ```
 
 ## Use Chronos
-A conda environment is created for you automatically. `bigdl-chronos` and the necessary depenencies (based on the build args) are installed inside this environment.
+A conda environment is created for you automatically. `bigdl-chronos` and the necessary depenencies (based on the build args when you build image) are installed inside this environment.
 ```bash
 (chronos) root@icx-5:/opt/work# 
 ```
```

Comment:
following highlight has an invalid link: `[Build an image](#build-an-image-optional)`

Reply:
> following highlight has an invalid link: `[Build an image](#build-an-image-optional)`

fix, links in the guide are all available now.